### DISCUSSION TITLE
Please contact me

### DISCUSSION THREAD

**Discussion by robthide37** [OrcaSlicer/OrcaSlicer#643] [General]
@SoftFever Hey this rob from advanced3dprinting.com  I seeing if you would be interested In doing some work for me I'm looking for programmers and am willing to keep project open source and willing to pay for your time . If interested message me through discord or on my site https://discord.gg/XhAVKDNN

**Comment by robthide37** [comment]
If you arent interested i get it i just figured i would try and make sure you saw my message

**Reply by SoftFever** [reply]
Hi @robthide37 

Sorry for the late response. A bit busy with well everything...
Yeah, I currently have no plan to do commission work.
Thank you for contacting me.

**Comment by robthide37** [comment]
Ok no prob if you do change your mind let me know i would not care if you
used the stuff you did for me in your slicer or anything like that i just
more want to get things better for everyone while getting what me and a few
others need for machines we are developing . Like we have a customer
version of klipper that we are developing to run my A3dp Servos and would
like to integrate them at some point .
